crypto/evp/pmeth_gn.c

```diff
@@ -150,23 +150,53 @@ int EVP_PKEY_CTX_get_keygen_info(EVP_PKEY_CTX *ctx, int idx)
     return ctx->keygen_info[idx];
 }
 
+static EVP_PKEY *evp_pkey_new_key(int priv, int type, ENGINE *e,
+                                  const unsigned char *key, int keylen)
+{
+    EVP_PKEY_CTX *ctx = NULL;
+    EVP_PKEY *pkey = NULL;
+
+    /* Note for compatibility reasons we allow keylen to be < 0 in some cases */
```

Comment:
Put a comma or colon after the word Note

Reply:
Done